src/benchmarks/micro/libraries/System.Text.Json/Utf8JsonReader/Perf.Get.cs

```diff
@@ -0,0 +1,265 @@
+        public sbyte GetSByte()
+        {
+            sbyte result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetSByte();
+            }
+            return result;
+        }
+
+        [Benchmark]
+        public short GetInt16()
+        {
+            short result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetInt16();
+            }
+            return result;
+        }
+
+        [Benchmark]
+        public int GetInt32()
+        {
+            int result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetInt32();
+            }
+            return result;
+        }
+
+        [Benchmark]
+        public long GetInt64()
+        {
+            long result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetInt64();
+            }
+            return result;
+        }
+
+        [Benchmark]
+        public ushort GetUInt16()
+        {
+            ushort result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetUInt16();
+            }
+            return result;
+        }
+
+        [Benchmark]
+        public uint GetUInt32()
+        {
+            uint result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetUInt32();
+            }
+            return result;
+        }
+
+        [Benchmark]
+        public ulong GetUInt64()
+        {
+            ulong result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetUInt64();
+            }
+            return result;
+        }
+
+        [Benchmark]
+        public float GetSingle()
+        {
+            float result = 0;
+            var reader = new Utf8JsonReader(_jsonNumberBytes);
+            reader.Read();
+            while (reader.Read() && reader.TokenType == JsonTokenType.Number)
+            {
+                result += reader.GetSingle();
```

Comment:
the input for `GetSingle` and `GetDouble()` should be sth more than `0.0` or `123.0`. Perhaps `123.456` would be a better number? Just to make sure it performs work that is closer to real-life scenarios